package.json

```diff
@@ -31,11 +31,10 @@
     "example": "ts-node -O '{ \"module\": \"commonjs\", \"target\": \"es2019\" }'",
     "view": "yarn api-explorer",
     "wipe": "rm -rf api spec",
-    "build": "yarn prepare",
-    "prepare": "rm -Rf packages/*/lib && run-p -c prepare:*",
-    "prepare:cjs": "lerna exec --stream 'BABEL_ENV=build_cjs babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments'",
-    "prepare:es": "lerna exec --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib/esm --source-maps --extensions .ts,.tsx --no-comments'",
-    "prepare:ts": "lerna exec --stream --sort 'tsc -b tsconfig.build.json'",
+    "build": "rm -Rf packages/*/lib && run-p -c build:*",
+    "build:cjs": "lerna exec --stream 'BABEL_ENV=build_cjs babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments'",
+    "build:es": "lerna exec --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib/esm --source-maps --extensions .ts,.tsx --no-comments'",
+    "build:ts": "lerna exec --stream --sort 'tsc -b tsconfig.build.json'",
```

Comment:
Looks good to me.

Reply:
FYI I copied the `"clean"` script into the `"build"` script because I'm trying to avoid calling `yarn` from inside any script that needs to run as part of the github action publish flow since it's using npm (due to wombat dressing room compatibility issues).

One way to DRY that out would be to create a `clean.sh` executable script and then call `./clean.sh` in both `"build"` and `"clean"`